pages/politicians/[slug].tsx

```diff
@@ -268,7 +268,7 @@ const PoliticianPage: NextPage<{ mobileNavTitle?: string }> = ({
           <Scroller onePageAtATime>
             {edges
               .map((edge) => {
-                return <BillCard bill={edge.node} key={edge.node.slug} />;
+                return <BillCard bill={edge.node} key={edge.node.slug} itemId={edge.node.slug} />;
```

Comment:
@wileymc the `itemId` is a prop that all children of the horizontal scroller need set for the scroller to work properly. I had added `itemId` as a prop to the necessary components like `BillCard` here because I was seeing this error on build. 

```
Failed to compile.
--
08:25:21.649 |  
08:25:21.649 | ./pages/politicians/[slug].tsx:271:72
08:25:21.650 | Type error: Type '{ bill: Partial<BillResult>; key: string \| undefined; itemId: string \| undefined; }' is not assignable to type 'IntrinsicAttributes & { bill: Partial<BillResult>; }'.
08:25:21.650 | Property 'itemId' does not exist on type 'IntrinsicAttributes & { bill: Partial<BillResult>; }'.
08:25:21.650 |  
08:25:21.650 | 269 \|             {edges
08:25:21.650 | 270 \|               .map((edge) => {
08:25:21.651 | > 271 \|                 return <BillCard bill={edge.node} key={edge.node.slug} itemId={edge.node.slug} />;
```
However it looks like the linter removed it as an unused prop and now I am seeing that error again. Any thoughts on how to proceed here?

Reply:
check out the commit I pushed.  that was my bad I saw that itemId was not being used so I removed in from the billCard props.  Its now being passed to the containing div in bill card, so i think this should do the trick.  does look like there are still some other tsc errors though